HGAlibs/HGAlibs2.py

```diff
@@ -0,0 +1,33 @@
+""" The Healing Gem of Arcadia Mad Libs Generator """
+
+word1 = input("Enter a word: ")
+pluralNoun = input("Enter a plural noun: ")
+word2 = input("Enter a second word: ")
+place = input("Enter your favorite place: ")
+adjective = input("Enter an adjective: ")
+word3 = input("Enter a third word: ")
+
+print ("----------YOUR HEALING GEM OF ARCADIA LIBS---------")
+
+print ("The healing gem and the",word1,"went along with", pluralNoun)
+
+print ("Because a healing gem is no use without", word2,",")
+
+print ("Remember the healing gem",pluralNoun,"in",place,".")
+
+print ("Where the Arcadia weather is always",adjective,".")
+
+print ()
+
+print ("You may think that The Healing Gem of Arcadia is",word3,",")
+
+print ("And you might be correct. But if you listen deeply you'll really hear its magic.")
+
+print ("------------------------------------------")
+
+print ("------------THANKS FOR PLAYING!!!---------")
+
+print ("------------------------------------------")
+
```

Comment:
Would look into a formatter. black is very popular: https://black.readthedocs.io/en/stable/getting_started.html